tensorflow/contrib/learn/python/learn/estimators/estimator.py

```diff
@@ -400,8 +400,8 @@ def __init__(self, model_dir=None, config=None):
 
     # Features and labels TensorSignature objects.
     # TODO(wicke): Rename these to something more descriptive
-    self._features_info = None
-    self._labels_info = None
+    self._features_info = {}
+    self._targets_info = {}
```

Comment:
@martinwicke - I think you are looking at previous version of the file. It was already renamed after in my last commit.